packages/@lwc/module-resolver/scripts/jest/types.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+export {}; // required to have a module with just `declare global` in it
+
+declare global {
+    namespace jest {
+        interface Matchers<R> {
+            __type: R; // unused, but makes TypeScript happy
+            toThrowErrorWithCode(received: any, ctor: any, message?: string): CustomMatcherResult;
+            toThrowErrorWithType(received: any, ctor: any, message?: string): CustomMatcherResult;
+        }
+    }
+}
```

Comment:
This should be in a `.d.ts` file to avoid having to export something.

Reply:
It doesn't seem to work. I have this in the `.d.ts` file:

```ts
declare namespace jest {
    interface Matchers<R> {
        __type: R; // unused, but makes TypeScript happy
        toThrowErrorWithCode(received: any, ctor: any, message?: string): CustomMatcherResult;
        toThrowErrorWithType(received: any, ctor: any, message?: string): CustomMatcherResult;
    }
}
```

 ...but the file importing it still complains about `Property 'toThrowErrorWithCode' does not exist on type 'JestMatchers() => RegistryEntry>'.`. Is there something I'm missing?